incubator/virtualcluster/pkg/controller/virtualcluster/master_provisioner_native.go

```diff
@@ -304,23 +305,28 @@ func (mpn *MasterProvisionerNative) createPKI(vc *tenancyv1alpha1.VirtualCluster
 	}
 	caGroup.APIServer = apiserverCAPair
 
-	finalAPIAddress := apiserverDomain
+	finalAPIDomain := apiserverDomain
 	if clusterIP != "" {
-		finalAPIAddress = clusterIP
+		if strings.Contains(clusterIP, ":") {
```

Comment:
We probably want to make this slightly more robust as well and use something like:

```
import "net"

// ...

dr := net.ParseIP("2001:db8::68")
addr := net.ParseIP("192.0.2.1")

if addr.To4() != nil {
	// is ipv4
} else {
	// is ipv6
}
///
```

Reply:
corrected, thanks